src/appkit.cs

```diff
@@ -13055,6 +13323,43 @@ interface NSSegmentedControl {
 		[Mac (10, 12, 2)]
 		[NullAllowed, Export ("selectedSegmentBezelColor", ArgumentSemantic.Copy)]
 		NSColor SelectedSegmentBezelColor { get; set; }
+
+		[Mac (10,13)]
+		[Export ("setToolTip:forSegment:")]
+		void SetToolTip ([NullAllowed] string toolTip, nint segment);
+
+		[Mac (10,13)]
+		[Export ("toolTipForSegment:")]
+		[return: NullAllowed]
+		string GetToolTip (nint forSegment);
+
+		[Mac (10,13)]
+		[Export ("setTag:forSegment:")]
+		void SetTag (nint tag, nint segment);
+
+		[Mac (10,13)]
+		[Export ("tagForSegment:")]
+		nint GetTag (nint segment);
+
+		[Mac (10,13)]
+		[Export ("setShowsMenuIndicator:forSegment:")]
+		void SetShowsMenuIndicator (bool showsMenuIndicator, nint segment);
+
+		[Mac (10,13)]
+		[Export ("showsMenuIndicatorForSegment:")]
+		bool GetShowsMenuIndicator (nint segment);
```

Comment:
mmm I do not quite like this name but can't come up with something better, not sure if removing Get would be better.